File: restorent backend/check.js
const sqlite3 = require('sqlite3').verbose();
const db = new sqlite3.Database('c:/manoj/backend/restaurant-pos-backend/restaurant.db');
db.all('SELECT * FROM collections WHERE collection_name="kitchens"', (err, rows) => {
  if (err) throw err;
  console.log("KITCHENS:");
  console.log(rows.map(r => JSON.parse(r.document).branch_name));
  db.close();
});

File: restorent backend/check.py
import sqlite3
import json
conn = sqlite3.connect('restaurant.db')
c = conn.cursor()
c.execute("SELECT json_extract(document, '$.branch_name'), json_extract(document, '$.company_name'), json_extract(document, '$.branch') FROM collections WHERE collection_name='kitchens'")
print(c.fetchall())
conn.close()
